tutorials/btp-hana-tenantdb-xsapp-create/btp-hana-tenantdb-xsapp-create.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Create an SAP HANA Tenant Database and an SAP HANA XS Classic Application
+description: Create and bind an SAP HANA tenant database to an SAP HANA XS classic application.
+auto_validation: true
+time: 15
+tags: [ tutorial>beginner, topic>Cloud, products>sap-business-technology-platform, tutorial>license]
+primary_tag: products>sap-hana-service-for-sap-btp
+author_name: Sylvia Hördt
+author_profile: https://github.com/sapsy
+---
+
+## Prerequisites
+ - You have a subaccount in the SAP BTP, Neo environment.
+ - You're assigned the Administrator role for the subaccount.
```

Comment:
Please change to "You are"

Reply:
This contracted form is aligned with acrocheck, our spellcheck tool for DITA CMS. Why do you want me to change it?